src/vs/platform/sharedProcess/node/sharedProcessEnvironmentService.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { memoize } from 'vs/base/common/decorators';
+import { Schemas } from 'vs/base/common/network';
+import { URI } from 'vs/base/common/uri';
+import { NativeEnvironmentService } from 'vs/platform/environment/node/environmentService';
+
+export class SharedProcessEnvironmentService extends NativeEnvironmentService {
+
+	@memoize
+	override get userRoamingDataHome(): URI { return this.appSettingsHome.with({ scheme: Schemas.userData }); }
```

Comment:
Is this applying to all settings? I am not entirely user what "user roaming data" is?

Reply:
yes - this is home for all user data - settings, keybindings, snippets. This is same as in workbench - 

https://github.com/microsoft/vscode/blob/33f84b36fd6757d37d3c8f5e2da0653b371641ec/src/vs/workbench/services/environment/electron-sandbox/environmentService.ts#L60